server/routes/user.py

```diff
@@ -42,3 +42,17 @@ def get_user(username):
         except ResourceDoesNotExistError:
             return error(f"User '{username}' not found.", 404)
 
+@user_bp.route("/<name>", methods=['POST'])
```

Comment:
We do not have any way to update the users yet, but when we it will be using this route. You'll have to make it route like `/<name>/add-assignment', also with a POST method.

Reply:
JK delete this